File: atinject-parent/atinject-core/src/main/java/org/atinject/core/logging/LoggerFactory.java
package org.atinject.core.logging;

import org.slf4j.Logger;

public class LoggerFactory
{
    private LoggerFactory()
    {
        
    }
    
    public static Logger getLogger(Class<?> clazz){
        return org.slf4j.LoggerFactory.getLogger(clazz);
    }
    
    public static Logger getLogger(String name){
        return org.slf4j.LoggerFactory.getLogger(name);
    }
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/LoggerFactory.js
define(["atinject/Logger"],
function(Logger) {
	function LoggerFactory(Logger Logger) {
		this.Logger = Logger;
	};

	LoggerFactory.prototype.getLogger = function(name) {
		// TODO get logger configuration
		return new Logger(Logger.INFO_LEVEL);
	};

	return new LoggerFactory();
});
